packages/api/test/scripts/helpers.js

```diff
@@ -52,3 +55,13 @@ export async function updateNameRecord (privKey, existingRecord, newValue) {
 export function getTestJWT (sub = 'test-magic-issuer', name = 'test-magic-issuer') {
   return JWT.sign({ sub, iss: JWT_ISSUER, iat: 1633957389872, name }, SALT)
 }
+
+/**
+ * @param {number} code
+ * @returns {Promise<string>}
+ */
+export async function randomCid (code = pb.code) {
+  const bytes = crypto.randomBytes(10)
```

Comment:
Minor thing, but this is using a cryptographically secure random algorithm, which isn't really necessary for tests and will probably be slow. If it's called a lot of times then it might be worth swapping this out for a more basic thing using `Math.random()`, just to speed things up.